ts/components/profile/ProfileScreenComponent.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import {
+  StyleSheet,
+} from "react-native";
+import { View } from "native-base";
+import { SvgProps } from "react-native-svg";
+import ListItemComponent from "../../components/screens/ListItemComponent";
+
+const style = StyleSheet.create({
+    listItem: {
+      flex: 1,
+      flexDirection: "row",
+    },
+    iconItem: {
+      flex: 1,
+      marginEnd: 16,
+      marginStart: 8,
+    },
+    textSection: {
+      flex: 1,
+      flexDirection: "column",
+    }
+  });
+  const iconProps = { width: 28, height: "auto" };
+
+// TODO: manage profile icons and study flex (see if there is a standard IO Theme
+export const ProfileListComponent = (props: {
```

Comment:
Why the name of the file is `ProfileScreenComponent` while the component is `ProfileListComponent` ?

Reply:
Good catch! It's a typo! 🙌